tests/lang/tests/var/var_err.yaiwr

```diff
@@ -0,0 +1,6 @@
+// Run-time:
+//    stdout:
+//    stderr: Evaluation error: Cannot reference undeclared variable by 'a' id!
+
+a = 1;
+rintln(a);
```

Comment:
Is that a typo? Did this compile?

Reply:
yes, fixed :point_right: 509bd2e8f57ccf101c2d5f66b7a89519b91f3fd6
I think it did pass the parsing cause it treated it as a  function call, but it failed at line 5 when evaluated